common/main/java/com/couchbase/lite/internal/BaseImmutableReplicatorConfiguration.java

```diff
@@ -181,12 +187,26 @@ public void addConnectionOptions(@NonNull Map<String, Object> options) {
         final Map<String, Object> httpHeaders = new HashMap<>();
         // User-Agent:
         httpHeaders.put("User-Agent", CBLVersion.getUserAgent());
+
         // headers
-        if ((headers != null) && (!headers.isEmpty())) {
+        // If there are cookies, we add them in options as
+        // REPLICATOR_OPTION_COOKIES instead of REPLICATOR_OPTION_EXTRA_HEADERS
+        if (headers != null) {
+            final String customCookies = headers.remove(AbstractCBLWebSocket.HEADER_COOKIES);
```

Comment:
Has this HEADER_COOKIE been defined in AbstractCBLWebSocket yet? I don't see it.

Reply:
It's in master branch but not in andriod/release/3.0 branch.

I'm not sure if I'm supposed to review the code but I am a reviewer on another PR that depends on this PR.

Reply:
That is correct.  I don't know of any way to make a PR from one repo depend on another.

Reply:
@LaurenNguyen14 What change are you requesting?

Reply:
@bmeike  This variable AbstractCBLWebSocket.HEADER_COOKIES are not defined yet in this PR. This branch doesn't compile in my local because of that variable. However, I remembered noticing it is defined in a different PR. So as long as both of these PRs are merged, it would be fine. I'll go ahead and approve.

Reply:
Perfect.  Thanks.